### DISCUSSION TITLE
C语言代码提示，需要安装什么插件呢？

### DISCUSSION THREAD

**Discussion by tommy0906i** [opensumi/core#2052] [General]
C语言怎么做代码提示，需要安装什么插件呢？

**Comment by erha19** [comment]
@tommy0906i  可以用这个插件：

![image](https://user-images.githubusercontent.com/9823838/205870033-24aedd44-8c49-4055-a8d1-05cc994e8656.png)

**Reply by tommy0906i** [reply]
docker版本的搜不到这个插件
 ![image](https://user-images.githubusercontent.com/119069785/205879306-1a9aec04-e7ce-48e9-ae86-f835fffe307e.png)

**Reply by tommy0906i** [reply]
更新点击以后安装不了

**Reply by erha19** [reply]
@tommy0906i 可以自行升级一下插件版本，示例仓库内的 Docker 镜像仅为快速上手使用，需要自行维护版本，无法更新可能为插件市场镜像访问缓慢导致，可以根据配置文档进行切换 https://opensumi.com/zh/docs/integrate/universal-integrate-case/custom-marketplace-entrypoint
